packages/manager/apps/vrack-services/e2e/utils/network.ts

```diff
@@ -0,0 +1,14 @@
+import { BrowserContext } from '@playwright/test';
+import { ICustomWorld } from '@playwright-helpers';
+import { toPlaywrightMockHandler } from '../../../../../../playwright-helpers';
```

Comment:
```suggestion
import { ICustomWorld, toPlaywrightMockHandler } from '@playwright-helpers';
```
If it's same path / alias for import

Reply:
aliases don't work in playwright